tests/composite_pk/models/generic.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib.contenttypes.fields import GenericRelation
+from django.db import models
+
+
+class Dummy(models.Model):
+    pk = models.CompositePrimaryKey("int", "datetime", "date", "uuid", "char")
+    int = models.IntegerField()
+    datetime = models.DateTimeField()
+    date = models.DateField()
+    uuid = models.UUIDField()
+    char = models.CharField(max_length=5)
```

Comment:
Yes, this is an issue... we'll need to warn users in the docs, not sure if it can be addressed

Reply:
I improved the docs, please check!

I think since we mention we serialize the PK into JSON, it's obvious it won't work if the field cannot be serialized into JSON.

I don't know of any use case where this might be the case though.